### DISCUSSION TITLE
Please add a simple Tutorial for vercel depolyment.

### DISCUSSION THREAD

**Discussion by SkysCrystal** [wangchucheng/hugo-eureka#154] [Q&A]
I've been stuck when deploying it on Vercel.
 
When I start a Hugo example in Vercel and replace all the repository files with a local blog folder which performed well on the local computer. And Vercel site just gives a 404 page.

I think that maybe the file `config.yaml` isn't in the root folder. So deployments may need more commands. I don't know how to fix it. 

I want to use this template and use notion as a CMS. Please give me some help. Thanks a lot!

**Comment by wangchucheng** [comment]
I think if you want to deploy to Vercel, you just need to install go in environment. Vercel did not contain go by default. Maybe we can just convert it to a discussion where is the best place to look for help.

**Reply by SkysCrystal** [reply]
谢谢回复！

我本身不是很懂技术，在B站上找了几个视频，最后采用在本地hugo好生成博客，用vercel分享public文件夹的办法。
之后我再找找教程，能不能上传文件后，让vercel自动生成hugo。

用notion做CMS，我找的 https://github.com/lvntbkdmr/NotiGoCMS 项目，自己能力有限，实现不了，也只能等之后再试试了。或者有机会能看到谁的教程学习一下。
